src/contexts/PlaidContext.res

```diff
@@ -0,0 +1,18 @@
+// type paymentScreenType = PAYMENTSHEET | SAVEDCARDSCREEN
```

Comment:
remove this context as this is not being used.

Reply:
sure

Reply:
removed 👍